docs/css-prop.mdx

```diff
@@ -31,7 +31,7 @@ Use the [JSX Pragma](#jsx-pragma) method instead.
   "presets": ["@emotion/babel-preset-css-prop"]
 }
 ```
-
+Compatibility Note: If you include the plugin `babel-plugin-transform-react-inline-elements` in your `.babelrc` your styles will not be applied. The plugin is not compatible with the `css` prop.
```

Comment:
I'm not so sure about the change - it obviously doesn't hurt anybody, but I'm wondering if it's the best place for such information. @mitchellhamilton wdyt?

Reply:
I've decided not to include it here as it's already mentioned on the bottom - **but** I've added a new warning about this to the `@emotion/babel-preset-css-prop`'s README